src/main.rs

```diff
@@ -94,26 +103,46 @@ pub fn main() -> Result<(), Box<dyn Error>> {
 
     let clang = Clang::new()?;
     let index = Index::new(&clang, false, false);
-
-    let tu = BaoTU::from(
-        index
-            .parser(source)
-            .arguments(if pe.is_64 {
-                &["-Werror"]
-            } else {
-                &["-m32", "-Werror"]
-            })
-            .parse()?,
-    );
-
-    let mut generated = pdb_wrapper::PDB::new(false)?;
+    let mut args = Vec::new();
+    let coptions = matches.values_of("coptions");
+    if coptions.is_some() {
+        let copts= coptions.unwrap();
+        for coption in copts {
+            let unquoted = coption.replace("\"","");
+            args.push(unquoted);
+        }
+    }
+    if pe.is_64 {
+        //args.push(String::from("-Werror"));
```

Comment:
Please don't comment out code, just remove it if you don't think it's necessary. Why are you removing -Werror anyway?

Reply:
i commented it out because it was your original code and i added a check to distinguish between warning and error

Reply:
I went with -Werror because the compiler does not cover the entirety of what is possible when defining structs in C(++) and even less so any possible extensions. I don't remember a specific case, I mostly just remember that I had it in as an anti-footgun.